utils.py

```diff
@@ -20,33 +35,42 @@ def parse_long_dates(date_string):
         month (str): The month as a string representing an integer between 1 and 12
         day (str): The day as a string representing an integer between 1 and 31
     """
+
+
+    date_regex = re.compile(r'([\w]*)\s+(\d{1,2})\D*(\d{4})', re.IGNORECASE)
```

Comment:
OK. This is a really helpful example of returning a number of matching groups from the regex. Noted!